pkgs/top-level/python-packages.nix

```diff
@@ -7150,6 +7150,8 @@ self: super: with self; {
     callPackage ../development/python-modules/jupyter-nbextensions-configurator
       { };
 
+  jupyterlab-vim = callPackage ../development/python-modules/jupyterlab-vim { };
```

Comment:
This should be below `jupyterlab-server` to keep the ordering

Reply:
Sorry that the CI log is such a mess. I have submitted #394131 to make it nicer.